### DISCUSSION TITLE
Symfony/Messenger: Supervisor

### DISCUSSION THREAD

**Discussion by MohamedZogh** [symfony/symfony#43991] [Q&A]
Hello,

I have set up a supervisor for my queue system, but despite the correct configuration (See https://symfony.com/doc/current/messenger.html#supervisor-configuration), the supervisor receives the tasks but they are never executed by my server. Do you have an idea ?
Thank you

**Comment by michaoj** [comment]
Did you start your worker ? Do you see it running in supervisorctl ?

**Reply by MohamedZogh** [reply]
Yes i have start and it running

**Reply by michaoj** [reply]
Can you try ton run the consume command you configured in supervisor in verbose mode from the machine where supervisor is running ? This would help to diagnose the problem
